docs/pages/dev/reference/API/org_unit_change_request_configuration.md

```diff
@@ -0,0 +1,240 @@
+      "editable_fields": "Array<String> - List of possible fields",
+      "possible_parent_type": [
+        {
+        "id": "Int - OrgUnit Type ID",
+        "name": "String - OrgUnit Type name"
+        }
+      ],
+      "possible_group_set_ids": [
+        {
+          "id": "Int - Group ID",
+          "name": "String - Group name"
+        }
+      ],
+      "editable_reference_form_ids": [
+        {
+          "id": "Int - Form ID",
+          "name": "String - Form name"
+        }
+      ],
+      "created_at": "Timestamp",
+      "updated_at": "Timestamp"
+    }
+  ]
+}
+```
+
+#### 400 - Bad request
+
+- One or more of the parameters provided couldn't be parsed
+- One or more of the parameters provided couldn't matched to the related resource (E.g.: unknown project_id)
+- A parameter was given that is not recognized
+
+#### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+#### 403 - Forbidden
+
+- User doesn't have the proper permission to access this resource.
+
+
+
+## 3 - GET /api/change_requests_configs/{id}/
+
+API used to retrieve a Change Request configuration.
+
+### Permissions
+
+- Change Request Configuration
+
+### Possible responses
+
+#### 200 - OK
+
+```json
+{
+  "id": "Int - id in the database, I know Martin wants it",
+  "project_id": "Int - Project ID",
+  "project_name": "String - Project name",
+  "org_unit_type_id": "Int - OrgUnit Type ID",
+  "org_unit_type_name": "String - OrgUnit Type name",
+  "org_units_editable": "Boolean - Whether or not OrgUnits of this OrgUnit Type are editable",
+  "editable_fields": ["Array<String> - List of possible fields"],
+  "possible_parent_type": [
+    {
+    "id": "Int - OrgUnit Type ID",
+    "name": "String - OrgUnit Type name"
+    }
+  ],
+  "possible_group_set_ids": [
+    {
+      "id": "Int - Group ID",
+      "name": "String - GroupSet name"
+    }
+  ],
+  "editable_reference_form_ids": [
+    {
+      "id": "Int - Form ID",
+      "name": "String - Form name"
+    }
+  ],
+  "created_at": "Timestamp",
+  "updated_at": "Timestamp"
+}
+```
+
+#### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+#### 403 - Forbidden
+
+- User doesn't have the proper permission to access this resource.
+
+#### 404 - Not found
+
+- The given ID was not found
+
+
+## 4 - GET /api/mobile/change_requests_configs/
+
+API used to list all Change Request configurations for the mobile app.
+
+### Permissions
+
+- User must be authenticated
+
+### Query parameters
+- app_id: String - Application ID for which to retrieve the configuration
+- page: Int (optional) - Current page (default: 1)
+- limit: Int (optional) - Number of entities returned by page (default: 20)
+
+### Possible responses
+
+#### 200 - OK
+
+```json
+{
+  "count": "Long",
+  "has_next": "Boolean",
+  "has_previous": "Boolean",
+  "page": "Long",
+  "pages": "Long",
+  "limit": "Long",
+  "results": [
+    {
+      "org_unit_type_id": "Int - OrgUnit Type ID",
```

Comment:
expose perhaps uuid or id ?

Reply:
I still don't understand why there is an `UUID` field but that's up to you.
Regarding the ID, this is an implementation detail from the backend (IMHO, it shouldn't have one and be a composite key on `project_id` and `org_unit_type_id), therefore there is no need to return it to the mobile.